cmd/list/upgrade/cmd.go

```diff
@@ -84,15 +97,34 @@ func run(_ *cobra.Command, _ []string) {
 		if notes == "" && (i == 0 || availableUpgrade == latestRev) {
 			notes = "recommended"
 		}
-		if availableUpgrade == scheduledUpgrade.Version() {
-			notes = fmt.Sprintf("%s for %s", upgradeState.Value(),
-				scheduledUpgrade.NextRun().Format("2006-01-02 15:04 MST"))
+		if !isHypershift {
+			notes = formatScheduledUpgrade(availableUpgrade, scheduledUpgrade, notes, upgradeState)
+		} else {
+			notes = formatScheduledUpgradeHypershift(availableUpgrade, controlPlaneScheduledUpgrade, notes)
```

Comment:
Just leaving this as a note for myself, I think we need to come up with a different way of breaking up flows like this between hypershift and classic. Try and flatten out the code and make it easier to read. 

Not a blocker for merging this as it is the common pattern all over the CLI

Reply:
+1 on that, we need a ticket to track this work